src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -867,11 +868,21 @@ class AccessionStore(
                     1)
                 .else_(0))
 
+    val speciesQuery =
+        dslContext
+            .select(DSL.countDistinct(ACCESSIONS.SPECIES_ID))
+            .from(ACCESSIONS)
+            .innerJoin(SPECIES)
+            .on(ACCESSIONS.SPECIES_ID.eq(SPECIES.ID))
+            .and(SPECIES.DELETED_TIME.isNull)
+            .and(ACCESSIONS.ID.`in`(subquery))
+
+    val speciesCount = speciesQuery.fetchOne()?.value1()
```

Comment:
May as well inline `speciesQuery` here. The reason the existing query in this function is stored in a variable is so we can log it and then execute it, but we're not logging this query.